geonode_mapstore_client/client/js/components/MediaViewer/PdfViewer.jsx

```diff
@@ -3,12 +3,12 @@ import Loader from '@mapstore/framework/components/misc/Loader';
 
 import { getFileFromDownload } from '@js/utils/FileUtils';
 
-const PdfViewer = ({src, embedUrl}) => {
+const PdfViewer = ({src, embedUrl, isExternalSource}) => {
```

Comment:
We should use only src as per issue description and not the embedUrl

Reply:
Thanks. I misinterpreted it